### PR TITLE
Fixes #27080 - pulp3 not in ping if disabled

### PR DESCRIPTION
**Pull request by parthaa**
Pulp3 status information will not show up in ping if pulp3 is disabled
on the smart proxy. Check the issue for the steps.

### CODE REVIEW

File: app/models/katello/ping.rb [hunk 1]
```diff
@@ -13,6 +13,10 @@ class << self
       def ping(services: SERVICES, capsule_id: nil)
         result = {}
         services.each { |service| result[service] = {} }
+        if capsule_id && SmartProxy.find(capsule_id).pulp3_enabled?
```
Comment (jlsherrill) [inline app/models/katello/ping.rb]: could capsule_id be nil here?  (Should you default to master if it is?

File: app/models/katello/concerns/smart_proxy_extensions.rb [hunk 2]
```diff
@@ -136,6 +136,10 @@ def backend_service_type(repository)
         end
       end
 
+      def pulp3_enabled?
+        !self.capabilities('Pulp3').nil?
```
Comment (jlsherrill) [inline app/models/katello/concerns/smart_proxy_extensions.rb]: This isn't the best way to test, i think has_feature?('Pulp3')  is better.  What you're doing is fetching the capabiltiies and checking if its nil, which is kind of a side effect of it not having the pulp3 feature

File: app/models/katello/ping.rb (hunk 1 above)
Comment (parthaa) [reply app/models/katello/ping.rb]: Yes. It can be null. ```curl -k http://localhost:3000/katello/api/ping``` had nil for capsule

**Comment by parthaa** [comment]
[test katello]

**Pull request merged**